coalib/bears/requirements/PipRequirement.py

```diff
@@ -33,11 +31,11 @@ def install_command(self):
         """
         Creates the installation command for the instance of the class.
 
-        :param return: A string with the installation command.
+        :param return: A list with the installation command parameters.
         """
-        result = "{} install {}".format(self.MANAGER, self.package)
-        if self.version:
-            result += "=={}".format(self.version)
+        result = [sys.executable, '-m pip', 'install',
```

Comment:
im not 100% sure what your point is.

basically i should do ``'-m', 'pip'`` instead of ``'-m pip'`` right?